wondrous-app/components/Pod/members/styles.tsx

```diff
@@ -1,13 +1,223 @@
-import { Button, Typography } from '@mui/material';
+import Link from 'next/link';
```

Comment:
a lof of these components are identical to the ones from org/members? let's reuse